graphs/ag-sparql-federate.fbp

```diff
@@ -0,0 +1,51 @@
+# ag-sparql-federate.fbp
+
+# Executes a SPARQL query against a dynamic federation of AllegroGraph repositories
+
```

Comment:
Can you add a summary of the steps required to do a federated query or point to the documentation of an example?